재호/1406.py

```diff
@@ -0,0 +1,25 @@
+# https://www.acmicpc.net/problem/1406
+
+def solution():
+    stk1, stk2 = [], []
+    string = input()
+    for c in string:
+        stk1.append(c)
+    M = int(input())
+    for _ in range(M):
+        op = list(map(str, input().split()))
```

Comment:
```suggestion
        op = input().split()
```
`input()` 함수로 입력을 받으면 default로 string 형식으로 들어오기 때문에 string 변환이 필요없고, `split()` 함수 역시 리스트를 반환해서 추가적은 `list()` 함수 또한 제거 가능할 거 같아요!